cfgov/v1/auth_forms.py

```diff
@@ -36,14 +36,6 @@ class UserEditValidationMixin(PasswordValidationMixin):
     user_attribute = 'instance'
 
 
-class CFGOVUserCreationForm(UserEditValidationMixin, UserCreationForm):
-    pass
```

Comment:
@kave I'm hesitant to add that back in, as part of the point of this PR is we're going with the Django defaults.  Eventually I think we'll want to upgrade our Django version, where we can hook up the password policies much more seamlessly.